### ISSUE TITLE
Land disappearances

### ISSUE THREAD

**Issue by EllingDeehr** [MikkoVihtakari/ggOceanMaps#1]
Land masses seem to drop out "above" -55 degrees latitude when producing non-stereographic Antarctic maps.

A: `basemap(limits = c(-72, -35, -60, -40), rotate = TRUE)`

B: `basemap(limits = c(-72, -35, -57, -35), rotate = TRUE)`

Land masses disappear above -55 in A but not in B.

**Comment by MikkoVihtakari** [comment]
Hmm...this is because the `antarctic_land` object extends to around -50 latitude, not -30 as it should:

`sp::plot(antarctic_land)`

In A you plot the `antarctic_land` and in B you plot `dd_land`. A clear bug. The solution is to remake `antarctic_land` and update the `ggOceanMapsData` package. I will close this issue once it is done.

**Comment by EllingDeehr** [comment]
Perfect. Otherwise great package. Thanks Mikko.

**Comment by MikkoVihtakari** [comment]
This issue should be fixed in ggOceanMapsData commit c5fbb78a339cf6cb4f9a9aa5f4942c14edd0046c

``` r
library(ggOceanMaps)
#> Loading required package: ggplot2
#> Loading required package: ggspatial
#> Loading required package: ggOceanMapsData
basemap(limits = c(-72, -35, -60, -40), rotate = TRUE)
#> Using lon and lat as longitude and latitude columns, respectively.
#> projection transformed from +init=epsg:4326 to +init=epsg:3031
#> Using lon and lat as longitude and latitude columns, respectively.
#> projection transformed from +init=epsg:4326 to +proj=stere +lat_0=-90 +lat_ts=-71 +lon_0=-53.5 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs +ellps=WGS84 +towgs84=0,0,0
```

![](https://i.imgur.com/VJadC2C.png)

<sup>Created on 2020-06-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
